index.html

```diff
@@ -3,67 +3,91 @@
-        </ul>
-        <div class="hero__funds__raised__header">
-          <h2>Total contributions flowing into Oakland’s March 2020 Election:</h2>
-        </div>
-        <div class="hero__funds_raised__total">
-          {{ oakland_total_funds | dollars }}
-        </div>
+    <div class="hero__table-container">
+      <div>
+        <h2 class="hero__table__header">Oakland Election Live Snapshot</h2>
+        <table class="hero__table">
+            <thead class="hero__table__head">
+                <tr>
+                    <th></th>
+                    <th class="hero__column__header">MARCH 2020</th>
+                    <th class="hero__column__header">NOVEMBER 2020</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td class="hero__row__header">Contributions to date</td>
+                    <td class="hero__funds_raised__total">{{ oakland_march_total_funds | dollars }}</td>
+                    <td class="hero__funds_raised__total">{{ oakland_total_funds | dollars }}</td>
+                </tr>
+                <tr>
+                    <td class="hero__row__header">% of contributions from Oakland residents</td>
+                    <td class="hero__funds_raised__total">{{ oakland_march_residents_percentage | times: 100 | round }}%</td>
+                    <td class="hero__funds_raised__total">{{ oakland_residents_percentage | times: 100 | round }}%</td>
+                </tr>
+                <tr>
+                    <td class="hero__row__header">Top 3 third-party spenders (independent expenditures)</td>
+                    <td class="hero__top-spenders">
+                      <ul>
+                        <li>OUTFRONT MEDIA LLC</li>
+                        <li>Families and Educators for Public Education</li>
+                        <li>OAKLAND EDUCATION ASSOCIATION POLITICAL ACTION COMMITTEE</li>
+                      </ul>
+                    </td>
+                    <td class="hero__top-spenders">Info Coming Soon!</td>
+                </tr>
+            </tbody>
+        </table>
       </div>
     </div>
-    {{ hero-cta }}
   </div>
 </div>
-<div class="landing__description">
-  <h3 class="landing__description__header">
-    Open Disclosure was created to empower Oakland voters with timely, accurate, and useful information about the
-    role of money in local elections.
-  </h3>
+<div class="landing__description landing__alerts-signup">
+  <h2 class="landing__description__header">Sign up for daily campaign finance alerts.</h2>
+  <p class="landing__description__paragraph">
+    Receive email notifications when Oakland lobbyists, candidates, and elected officials disclose their finances.
+    Stay on top of the influence of money on your local politics with this useful daily digest.
+  </p>
+
+  <p class="landing__description__paragraph">
+    <strong><a href="https://alert.opendisclosure.io" class="btn landing__btn">Sign up for daily alerts</a></strong>
```

Comment:
Could we add a little more padding to this button? In the screenshots provided, the text looks pretty close to the borders and feels kinda cramped.